### ISSUE TITLE
[Windows] - Problems with linking Alby Account

### ISSUE THREAD

**Issue by itstomekk** [getAlby/hub#852]
I had two users that had this problem, and I can replicate on my machine.

When I have Alby Account not linked to any node, and try to link this (Windows) Alby Hub to my Alby Account, I'm (and other users) getting info that it couldn't be linked

![image](https://github.com/user-attachments/assets/989af8d5-00f8-476a-805d-13e2bacbd3f8)

In fact, on getalby.com/node it tells me that Alby Hub is linked

But the Hub still doesn't recognize it
![image](https://github.com/user-attachments/assets/b2e3b5a8-034d-4758-abb1-cffd1d3c02d7)

Only after refresh (and on Windows there is no 'refresh' button like in browser, so the only thing to check things is to shut the app down and open it again (another issue?), it says it is linked to another wallet

**Comment by itstomekk** [comment]
trying to send via this lightning address

![image](https://github.com/user-attachments/assets/73fb3220-bd7f-48ab-98ae-6f11f389f929)

**Comment by itstomekk** [comment]
Alby Dashboard shows the transactions (receive via invoice works ofc), but doesn't show the balance
![image](https://github.com/user-attachments/assets/4e0457d3-8d9e-40a5-8185-604b2130388c)

**Comment by itstomekk** [comment]
Creating the invoice via Alby Dashboard gives such error

![image](https://github.com/user-attachments/assets/35120344-6f04-4ed0-818d-766c0de137a9)

**Comment by itstomekk** [comment]
Sending to Lightning Address from Alby Dashboard works though

**Comment by jpl-btc** [comment]
The "Payment aborted: Recipient server error" is something that I have seen when the Alby web account is not linked to the Alby Hub.
